desc_dc2_dm_data/repos.py

```diff
@@ -16,4 +16,6 @@
     '2.1i_dr1a': '/global/cscratch1/sd/desc/DC2/data/Run2.1i/rerun/coadd-v1',
     '2.1i_dr1b': '/global/cscratch1/sd/desc/DC2/data/Run2.1i/rerun/coadd-dr1b-v1',
     '2.1i_dr4': '/global/cscratch1/sd/desc/DC2/data/Run2.1i/rerun/coadd-dr4-v1',
+    '2.2i_t3828': '/global/cscratch1/sd/desc/DC2/data/Run2.2i/rerun/run2.2-coadd-wfd-y1-v1',
```

Comment:
This should be '2.2i_dr1_tract3828'.  But I think it should just be '2.2i_dr1'.
This is where we're going to put everything in '2.2i_dr1' eventually, right?

Reply:
Sure, but it would be `2.2i_dr2` right?  As this isn't the first 6 month?  
My understanding about the processing is that y1+y2 coadds is coming next, so that will be `2.2i_dr3`.  I'm not currently anticipating that we'll have a full dr2 (y1 coadds) beyond just tract 3828... hence my thought to add in the tract in the name to hopefully make that clear.
Maybe we need to ask that CC utilize the `dr` in their rerun naming convention to make things more clear.

Reply:
I went ahead and changed this to `2.2i_dr2_tract3828`, but I'm open to other changes. I'm not anticipating we will have a full dr2.